src/nuclio/projects/project/functions/function-collapsing-row/function-collapsing-row.component.js

```diff
@@ -5,7 +5,6 @@
         .component('nclFunctionCollapsingRow', {
             bindings: {
                 actionHandlerCallback: '&',
-                createFunction: '&',
```

Comment:
Both Dashboard and Nuclio currently set this binding:
- https://github.com/nuclio/nuclio/blob/master/pkg/dashboard/ui/src/app/components/data-wrappers/functions-data-wrapper/functions-data-wrapper.tpl.html#L1
- https://github.com/iguazio/dashboard/blob/development/src/app/components/nuclio/functions-data-wrapper/functions-data-wrapper.tpl.html#L2

Instead of removing the above two lines from their repos, I think it's better to just revert the deletion of this line of code here, even though `nclFunctions` currently not uses this `createFunction` binding.